docs/src/main/docs/metrics/01_metrics.adoc

```diff
@@ -16,22 +16,25 @@
 
 ///////////////////////////////////////////////////////////////////////////////
 
-= Metrics and Health Support
-:description: Helidon Reactive WebServer metrics and health support
-:keywords: helidon, reactive, reactive streams, reactive java, reactive webserver, metrics
+= Metrics
+:description: Helidon metrics
+:keywords: helidon, metrics
 
 == Metrics Support
-Helidon WebServer includes support for MicroProfile Metrics. MicroProfile
-Metrics has a variety of metric types and two reporting formats that
-are exposed at the `/metrics` endpoint.
+Metrics support in Helidon SE provides:
+
+1. Endpoint `/metrics` - a configurable endpoint that exposes metrics information in JSON (as specified by
+    Microprofile Metrics specification), or plain text (as used by Prometheus metrics)
```

Comment:
1. The endpoint `/metrics`: A configurable endpoint that exposes metrics information in JSON format (as specified by the MicroProfile Metrics specification) or in plain text (for Prometheus metrics).